service/controller/v10/templates/cloudformation/guest/instance.go

```diff
@@ -28,10 +37,16 @@ const Instance = `{{define "instance"}}
       Tags:
       - Key: Name
         Value: {{ .Instance.Master.EtcdVolume.Name }}
-  {{ .Instance.Master.Instance.ResourceName }}MountPoint:
+  {{ .Instance.Master.Instance.ResourceName }}DockerMountPoint:
+    Type: AWS::EC2::VolumeAttachment
+    Properties:
+      InstanceId: !Ref {{ .Instance.Master.Instance.ResourceName }}
+      VolumeId: !Ref DockerVolume
+      Device: /dev/xvdc
+  {{ .Instance.Master.Instance.ResourceName }}EtcdMountPoint:
     Type: AWS::EC2::VolumeAttachment
     Properties:
       InstanceId: !Ref {{ .Instance.Master.Instance.ResourceName }}
       VolumeId: !Ref EtcdVolume
-      Device: /dev/sdh
+      Device: /dev/xvdh
```

Comment:
General: do you know where this `xvd` comes from?

Do both instance types support `xvd` devices, but only m3 supports `sd`?